### DISCUSSION TITLE
Select some in Pack of product with combinations in order to get a discount

### DISCUSSION THREAD

**Discussion by juraj1000** [PrestaShop/PrestaShop#40181] [Ideas]
#### Is your feature request related to a problem?
We'd like to use a pack of product with combinations where would be a possibility to select some single items from this pack in FO.

#### Describe the solution you'd like
We sell books and we want to create the pack of product with e.g. 5 items, these items are single books. We want to use discount if user will choose any 3 books from this pack. 
The functionality could be similar to single product with combinations where are used attributes for combinations. In our case the attributes would be single products. Also there should similar functionality as for attributes: define combinations, different prices for different combinations....

[a comment by an automated account is not shown]

**Comment by florine2623** [comment]
Hi @juraj1000 ,
Thanks for the suggestion.
Could you share the steps of that feature ?
example :
1. in BO, create pack of 5 products
2. in FO, customer chooses 3 products of the pack
3. etc

Can you also share screenshots of what it would look like in both BO and FO ?
Thanks!

**Comment by juraj1000** [comment]
_Example for pack with combinations_
- there are 7 books in the pack
- it is possible to select 3 books with discount 10 €

_Front Office_
- **Product details**
![image](https://user-images.githubusercontent.com/33311857/87149153-fe50e080-c2af-11ea-9a37-ae1285eed3fc.png)

_Back Office_

- **New product type "Pack with combinations"**
![image](https://user-images.githubusercontent.com/33311857/87147687-5803db80-c2ad-11ea-9d50-30014576f661.png)

- **Combinations**
combinations defined with Book 1, Book 2, Book 3
single product items Book 1.X, Book 2.X, Book 3.X 
![image](https://user-images.githubusercontent.com/33311857/87147823-9b5e4a00-c2ad-11ea-96ff-db5674585fa3.png)

- **Specific prices for pack**
![image](https://user-images.githubusercontent.com/33311857/87147859-aadd9300-c2ad-11ea-92f0-aaf7ce7c0c78.png)

**Comment by florine2623** [comment]
Hi @PrestaShop/prestashop-product-team ,

What do you think about that feature request ?
Thanks!

**Comment by colinegin** [comment]
I'm not sure I understand why you should have these books in a pack ? 
Why don't you use a catalog price rule stating that a discount is activated from 3 book units ?

**Comment by juraj1000** [comment]
Hello @colinegin,
there can be different combinations of books in the pack therefore I don't know how to define discount from 3 book units in this case.

_Example_
It is a little different as as it is shown in my previous screen shots (it was only example how it could be visible in BO)
- there are different single product prices
- there are 2 combinations but the customer can create also different combinations in his order

![image](https://user-images.githubusercontent.com/33311857/87530749-0c787580-c691-11ea-96a8-ee85edb6219e.png)
